src/main/java/com/minecraftabnormals/upgrade_aquatic/core/mixin/DrownedModelMixin.java

```diff
@@ -0,0 +1,68 @@
+                this.bipedLeftArm.rotateAngleY = this.rotLerpRad(leftArmSwimAnimTicks, this.bipedLeftArm.rotateAngleY, (float)Math.PI);
+                this.bipedRightArm.rotateAngleY = MathHelper.lerp(rightArmSwimAnimTicks, this.bipedRightArm.rotateAngleY, (float)Math.PI);
+                this.bipedLeftArm.rotateAngleZ = this.rotLerpRad(leftArmSwimAnimTicks, this.bipedLeftArm.rotateAngleZ, (float)Math.PI + 1.8707964F * this.getArmAngleSq(limbSwingRemainder) / this.getArmAngleSq(14.0F));
+                this.bipedRightArm.rotateAngleZ = MathHelper.lerp(rightArmSwimAnimTicks, this.bipedRightArm.rotateAngleZ, (float)Math.PI - 1.8707964F * this.getArmAngleSq(limbSwingRemainder) / this.getArmAngleSq(14.0F));
+             } else if (limbSwingRemainder >= 14.0F && limbSwingRemainder < 22.0F) {
+                float multiplier1 = (limbSwingRemainder - 14.0F) / 8.0F;
+                this.bipedLeftArm.rotateAngleX = this.rotLerpRad(leftArmSwimAnimTicks, this.bipedLeftArm.rotateAngleX, ((float)Math.PI / 2F) * multiplier1);
+                this.bipedRightArm.rotateAngleX = MathHelper.lerp(rightArmSwimAnimTicks, this.bipedRightArm.rotateAngleX, ((float)Math.PI / 2F) * multiplier1);
+                this.bipedLeftArm.rotateAngleY = this.rotLerpRad(leftArmSwimAnimTicks, this.bipedLeftArm.rotateAngleY, (float)Math.PI);
+                this.bipedRightArm.rotateAngleY = MathHelper.lerp(rightArmSwimAnimTicks, this.bipedRightArm.rotateAngleY, (float)Math.PI);
+                this.bipedLeftArm.rotateAngleZ = this.rotLerpRad(leftArmSwimAnimTicks, this.bipedLeftArm.rotateAngleZ, 5.012389F - 1.8707964F * multiplier1);
+                this.bipedRightArm.rotateAngleZ = MathHelper.lerp(rightArmSwimAnimTicks, this.bipedRightArm.rotateAngleZ, 1.2707963F + 1.8707964F * multiplier1);
+            } else if (limbSwingRemainder >= 22.0F && limbSwingRemainder < 26.0F) {
+                float multiplier2 = (limbSwingRemainder - 22.0F) / 4.0F;
+                this.bipedLeftArm.rotateAngleX = this.rotLerpRad(leftArmSwimAnimTicks, this.bipedLeftArm.rotateAngleX, ((float)Math.PI / 2F) - ((float)Math.PI / 2F) * multiplier2);
+                this.bipedRightArm.rotateAngleX = MathHelper.lerp(rightArmSwimAnimTicks, this.bipedRightArm.rotateAngleX, ((float)Math.PI / 2F) - ((float)Math.PI / 2F) * multiplier2);
+                this.bipedLeftArm.rotateAngleY = this.rotLerpRad(leftArmSwimAnimTicks, this.bipedLeftArm.rotateAngleY, (float)Math.PI);
+                this.bipedRightArm.rotateAngleY = MathHelper.lerp(rightArmSwimAnimTicks, this.bipedRightArm.rotateAngleY, (float)Math.PI);
+                this.bipedLeftArm.rotateAngleZ = this.rotLerpRad(leftArmSwimAnimTicks, this.bipedLeftArm.rotateAngleZ, (float)Math.PI);
+                this.bipedRightArm.rotateAngleZ = MathHelper.lerp(rightArmSwimAnimTicks, this.bipedRightArm.rotateAngleZ, (float)Math.PI);
+             } if (entityIn.isActualySwimming()) {
```

Comment:
These if and else statements are not aligned with the other lines in this else if statement.